src/sections/Pricing/Pricing.tsx

```diff
@@ -34,55 +39,120 @@ const pricingCards: PricingCardData[] = [
     title: 'Trial',
     price: 'Free',
     benefits: [
-      'Use it in PoC (Proof of Concept) to see how it fits your project.',
-      'Once ready to use it in your operations, purchase and enjoy it on production.',
+      {
+        icon: <CheckmarkIcon />,
+        text: 'Use it in PoC (Proof of Concept) to see how it fits your project.',
+      },
+      {
+        icon: <CheckmarkIcon />,
+        text: 'Once ready to use it in your operations, purchase and enjoy it on production.',
+      },
     ],
     buttonText: 'Try it out',
     buttonLink:
       'https://transactions.sendowl.com/products/78972481/998CD884/view',
     btnClassName: 'btn-trial',
   },
   {
-    color: '#DBDB40',
+    color: '#00ABFF',
     title: 'Monthly',
     price: '$149',
     denominator: 'month',
-    benefits: ["Forminer's complete package 💪"],
+    benefits: [
+      {
+        icon: <StarIcon />,
+        text: 'All features included.',
+        additionalStyles: {
+          backgroundColor: '#ECF0F1',
+          borderRadius: 32,
+          padding: 7,
+          transition: 'all 0.2s ease-out',
+        },
```

Comment:
This part of the code is repeated in 3 places. Let's put it into the `additionalStyles` variable above the `pricingCards` definition.